source/agora/test/ValidatorCleanRestart.d

```diff
@@ -31,10 +31,10 @@ import agora.test.Base;
 version(none) unittest
```

Comment:
Shouldn't this line be changed in order to re-enable the first test?

Reply:
Yes, thanks